docs/modules/ROOT/pages/index.adoc

```diff
@@ -228,24 +228,25 @@ This specification defines a _<<C2PA assertion>>_ known as an *<<_identity_asser
 
 The *<<_identity_assertion,identity assertion>>* contains the following data fields:
 
-* `referenced_assertions` contains a list of assertions that are referenced and signed by this _<<_credential_subject,credential subject>>_
+* `tbs` (“to be signed”) contains a set of data to be signed by this _<<_credential_subject,credential subject>>_. This structure contains:
+** `referenced_assertions` contains a list of other assertions selected by the credential subject. This is referred to as `tbs.referenced_assertions` throughout this specification.
 * `signature` contains the raw byte stream of the _<<_credential_subject,credential subject>>’s_ signature
 * `sig_type` defines the data type of signature
 * `pad1` and `pad2` are zero-filled byte strings used to fill space
```

Comment:
In the C2PA spec we are clear that this is `\0` and not "0"

Reply:
@lrosenthol since this isn't directly related to the `tbs` data structure, I split it off into a separate PR #79.